02-intermediate-ruby/object-inheritance.md

```diff
@@ -3,77 +3,138 @@
 
 ```ruby
-generic_bird = Bird.new
-generic_bird.talk #=> "I'm a bird. Kah."
+generic_item = Product.new
+generic_item.about #=> "12345: Generic Product"
 
-tux = Penguin.new
-tux.talk # => "I'm a bird. Kah."
+poodr = Book.new
+poodr.about #=> "12345: Generic Product"
 ```
 
-This is known as _implicit inheritance_ because when we put functionality in the _base class_ (`Bird`), then all _subclasses_ (`Penguin`) will automatically get those behaviors.
+This is known as _implicit inheritance_ because when we put functionality in the _base class_ (`Product`), then all _subclasses_ (`Book`) will automatically get those behaviors.
 
 ## Overriding Methods
 What happens when our classes have different functionality for the same idea? We can _override_ methods from _base classes_ in their _subclasses_.
 
 ```ruby
-class Penguin < Bird
-  def talk
-    "I'm a bird. Sqwaaa."
+class Book < Product
+  def about
+    return "#{id}: Book"
   end
 end
 
-generic_bird = Bird.new
-generic_bird.talk # => "I'm a bird. Kah."
+generic_item = Product.new
+generic_item.about #=> "12345: Generic Product"
 
-tux = Penguin.new
-tux.talk # => "I'm a bird. Sqwaaa."
+poodr = Book.new
+poodr.about # => "12345: Book"
 ```
 
-In this case, we override the functionality that the _base class_ (`Bird`) provides with more specific functionality in the _subclass_ (`Penguin`).
+In this case, we override the functionality that the _base class_ (`Product`) provides with more specific functionality in the _subclass_ (`Book`).
 
 ### Using `super` to invoke behavior in _base classes_.
 
-It's also possible to use the behavior defined in the _base class_ alongside specialized behavior in the _child class_. By using the keyword `super`, we can invoke the code in the _base class_ at any point in the _overriding_ method __and__ still be capable of using overriding code.
+It's also possible to use the behavior defined in the _base class_ alongside specialized behavior in the _child class_. By using the keyword `super`, we can invoke the code in the _base class_ at any point in the _overriding_ method __and__ still be capable of using overriding code.  In other words the overriding method can do the same action as the parent class **and** more.
+
+```ruby
+class AudioBook < Book
+  def about
+    # super invokes the `about` method on the _base class_ (Book)
+    return super + "-Audio"
+  end
+end
+
+generic_item = Product.new
+generic_item.about # => "12345"
+
+audiobook = AudioBook.new
+audiobook.about # => "12345: Book-Audio"
+```
+
+In this case, we utilize the behavior that the _base class_ (`Book`) provides but augment it with behavior specific to the _subclass_ (`AudioBook`).
+
+![super in method calls](images/super.png)
+
+## Super & Initialize
+
+We've seen how methods by themselves work with methods, what about instance variables?  Instance variables are inherited as well, for example:
```

Comment:
The section heading here talks about super and initialize, but the text talks about instance variables. This feels confusing to me.